test/Confluent.Kafka.IntegrationTests/Tests/ClosedHandle.cs

```diff
@@ -33,8 +35,9 @@ public static void Producer_ClosedHandle(string bootstrapServers, string singleP
             var producerConfig = new Dictionary<string, object>
             {
                 { "bootstrap.servers", bootstrapServers },
+                { "dotnet.producer.enable.background.poll", false }
             };
-            var producer = new Producer(producerConfig, true, false);
```

Comment:
great point, though 'tad' is an appropriate adjective since the methods all fall directly through to the same underlying implementation (and that hasn't changed). will do, don't think it'll take long.

Reply:
oh this is the constructor, this is important.